etna/transforms/math/statistics.py

```diff
@@ -445,6 +445,59 @@ def _aggregate_window(self, series: pd.Series) -> float:
         return tmp_series.median(**self.kwargs)
 
 
+class MADTransform(WindowStatisticsTransform):
+    """MADTransform computes Mean Absolute Deviation over the window."""
+
+    def __init__(
+        self,
+        in_column: str,
+        window: int,
+        seasonality: int = 1,
+        min_periods: int = 1,
+        fillna: float = 0,
+        out_column: Optional[str] = None,
+    ):
+        """Init MADTransform.
+
+        Parameters
+        ----------
+        in_column: str
+            name of processed column
+        window: int
+            size of window to aggregate
+        out_column: str, optional
```

Comment:
Many it would be better to put it in the last place

Reply:
fixed it in all the statistics Transforms 👌🏼